Tests/XcodeProjTests/Project/XCUserDataTests.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import PathKit
+import XCTest
+@testable import XcodeProj
+
+final class XCUserDataTests: XCTestCase {
+    func test_read_userData() throws {
+        let subject = try XCUserData(path: userDataPath)
+        assert(userData: subject, userName: "username1")
+    }
+
+    func test_write_userData() {
+        testWrite(from: userDataPath,
+                  initModel: { try? XCUserData(path: $0) },
+                  modify: { $0 },
+                  assertion: {
+                    assert(userData: $1, userName: "copy")
```

Comment:
`testWrite` copies the `username1.xcuserdatad` folder over to `<tmpDir>copy.xcuserdatad` thus changing the `userName` of the `XCUserData` to `copy`...